Felienne/hedy
# Reversed default customizations settings

**Description**
In this PR we change the structure of the class customizations. Currently be default alle levels and adventures are not selected, requiring the teacher to manually select these. To provide some additional feedback a red banner was show with a "no customizations yet" warning. We reverse this by setting all levels and adventures selected by default, making the banner redundant.

**Fixes**
This PR fixes #2224

**How to test**
Make sure you are a teacher and have a non-customized class (just create a new one). Navigate to the customizations screen and verify that all levels and adventures are selected by default. The functionality should be identical.
